CHANGELOG.md

```diff
@@ -2,11 +2,34 @@
 
 ## Unreleased
 
+## v0.9.0 Beta
+
 ## 🛑 Breaking changes 🛑
 
-- Remove `reconnection_delay` from OpenCensus exporter #1516.
-- Remove old receiver factories and receiver base factory #1583.
-- Remove logs factories and merge with normal factories #1569.
+- Remove `reconnection_delay` from OpenCensus exporter (#1516).
+- Remove old receiver factories and receiver base factory (#1583).
```

Comment:
I made PRs for all components not only for receivers

Reply:
Can we highlight this line? It is potentially the biggest breaking change for anyone who uses a custom build of Collector with custom components. It would be great to list actual interface names that are removed.